docs/get-started/tutorials/pulsar.md

```diff
@@ -0,0 +1,207 @@
+./quickwit index create --index-config stackoverflow.yaml
+```
+
+## Create and populate a Pulsar topic
+
+Now, let's create a Pulsar topic and send some messages into it.
+
+```bash
+# Execute these commands in the pulsar directory.
+bin/pulsar-admin tenants create quickwit
+# Check it worked.
+bin/pulsar-admin tenants list
+bin/pulsar-admin namespaces create quickwit/pulsar
+# Set retention policy for the namespace
+bin/pulsar-admin namespaces set-retention quickwit/pulsar --time 10m --size 1G
+# Create topic.
+bin/pulsar-admin topics create-partitioned-topic quickwit/pulsar/stackoverflow -p 4
+# Check it worked.
+bin/pulsar-admin topics list-partitioned-topics quickwit/pulsar
+```
+
+To populate the topic, we will use a python script:
+
+```python title=send_messages_to_pulsar.py
+import json
+import pulsar
+
+client = pulsar.Client('pulsar://localhost:6650')
+producer = client.create_producer('quickwit/pulsar/stackoverflow')
+
+with open('stackoverflow.posts.transformed-10000.json', encoding='utf8') as file:
+   for i, line in enumerate(file):
+       producer.send(line.encode('utf-8'))
+       if i % 100 == 0:
+           print(f"{i}/10000 messages sent.", i)
+
+client.close()
+```
+
+Install locally the python client, more details on [documentation page](https://pulsar.apache.org/docs/2.11.x/client-libraries-python/):
+
+```bash
+# Requires a python version < 3.11
+pip3 install 'pulsar-client==2.10.1'
+wget https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/stackoverflow/send_messages_to_pulsar.py
+python3 send_messages_to_pulsar.py
+```
+
+## Create the Pulsar source
+
+```yaml title="pulsar-source.yaml"
+#
+# Pulsar source config file.
+#
+version: 0.4
+source_id: pulsar-source
+source_type: pulsar
+params:
+  topics:
+    - stackoverflow
+  address: pulsar://localhost:6650
+```
+
+Run these commands to download the source config file and create the source.
+
+```bash
+# Download Pulsar source config.
+wget -O so-pulsar-source.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/stackoverflow/pulsar-source.yaml
+
+# Create source.
+./quickwit source create --index stackoverflow --source-config so-pulsar-source.yaml
+```
+
+As soon as the Pulsar source is created, Quickwit Control Plane will ask an indexer to start a new indexing pipeline. You can observe that by looking at the logs of indexers. On Kubernetes for example:
+
+## Time to search!
+
+You can run this command to inspect the properties of the index and check the current number of published splits and documents:
+
+```bash
+# Display some general information about the index.
+./quickwit index describe --index stackoverflow
+```
+
+You are now ready to execute some queries.
+
+```bash
+curl 'http://localhost:7280/api/v1/stackoverflow/search?query=search+AND+engine'
+```
+
+It is also possible to access these results through the [Quickwit UI](http://localhost:7280/ui/search?query=&index_id=stackoverflow&max_hits=10).
```

Comment:
I like the idea behind this, but if I'm a user having a read through this suggests to me I can click on the link to learn more about the UI rather than expecting it to take me to my localhost (maybe have a separate link for that?)